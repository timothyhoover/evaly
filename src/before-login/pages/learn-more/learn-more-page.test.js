import React from 'react';
import { render } from '@testing-library/react';
import LearnMorePage from './learn-more-page.component';

test('quiz renders correctly', () => {
	// arrange
	const component = render(<LearnMorePage />);

	// assert
	expect(component).toMatchSnapshot();
});
