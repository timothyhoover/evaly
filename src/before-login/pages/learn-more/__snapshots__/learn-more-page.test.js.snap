// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`quiz renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="learn-more-page"
      >
        <div
          class="main-section-learn"
        >
          <div
            class="main-section-learn__row-1"
          >
            <div
              class="main-section-learn__content"
            >
              <h1>
                Build Confidence
              </h1>
              <p>
                Discover what your English level and where to improve using our evaluation tools.
              </p>
              <button
                class="btn-secondary"
              >
                Get Started
              </button>
            </div>
            <img
              class="main-section-learn__img-1"
              src="learn-more-hero.png"
            />
          </div>
          <div
            class="main-section-learn__row-2"
          >
            <img
              class="main-section-learn__img-2"
              src="learn-more-main-1.png"
            />
            <div
              class="main-section-learn__content"
            >
              <h4>
                Evaluator
              </h4>
              <p>
                We evaluate your level based on real world exam situations that give an outcome close to what you might see on the Cambridge exam
              </p>
            </div>
          </div>
          <div
            class="main-section-learn__row-3"
          >
            <div
              class="main-section-learn__content"
            >
              <h4>
                Practice Exams
              </h4>
              <p>
                Take as many mock exams as you need to feel at ease on exam day. We pride ourselves at making the mock as encouraging but accurate as possible.
              </p>
            </div>
            <img
              class="main-section-learn__img-3"
              src="learn-more-main-2.png"
            />
          </div>
        </div>
        <div>
          <div
            class="banner-bottom-lm"
          >
            <div
              class="banner-bottom-lm__item"
            >
              <h2>
                Rock your English exams!
              </h2>
              <button
                class="btn-tertiary"
              >
                Get Started
              </button>
            </div>
          </div>
        </div>
        <div
          class="footer"
        >
          <div
            class="footer__item-1"
          >
            <p>
               A company by Topic Talk Inglés
            </p>
            <p>
              Copyright Evaly 2021 All rights reserved
            </p>
          </div>
          <div
            class="footer__item-2"
          >
            <p
              class="footer__logo"
            >
               Evaly
            </p>
            <svg
              class="icon footer__fb-icon"
            >
              <use
                href="sprite.svg#facebook"
              />
            </svg>
            <svg
              class="icon footer__instagram-icon"
            >
              <use
                href="sprite.svg#instagram"
              />
            </svg>
          </div>
          <div
            class="footer__item-3"
          >
            <ul>
              <li>
                <a
                  href="#"
                >
                  Terms & Conditions
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Privacy Policy
                </a>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="learn-more-page"
    >
      <div
        class="main-section-learn"
      >
        <div
          class="main-section-learn__row-1"
        >
          <div
            class="main-section-learn__content"
          >
            <h1>
              Build Confidence
            </h1>
            <p>
              Discover what your English level and where to improve using our evaluation tools.
            </p>
            <button
              class="btn-secondary"
            >
              Get Started
            </button>
          </div>
          <img
            class="main-section-learn__img-1"
            src="learn-more-hero.png"
          />
        </div>
        <div
          class="main-section-learn__row-2"
        >
          <img
            class="main-section-learn__img-2"
            src="learn-more-main-1.png"
          />
          <div
            class="main-section-learn__content"
          >
            <h4>
              Evaluator
            </h4>
            <p>
              We evaluate your level based on real world exam situations that give an outcome close to what you might see on the Cambridge exam
            </p>
          </div>
        </div>
        <div
          class="main-section-learn__row-3"
        >
          <div
            class="main-section-learn__content"
          >
            <h4>
              Practice Exams
            </h4>
            <p>
              Take as many mock exams as you need to feel at ease on exam day. We pride ourselves at making the mock as encouraging but accurate as possible.
            </p>
          </div>
          <img
            class="main-section-learn__img-3"
            src="learn-more-main-2.png"
          />
        </div>
      </div>
      <div>
        <div
          class="banner-bottom-lm"
        >
          <div
            class="banner-bottom-lm__item"
          >
            <h2>
              Rock your English exams!
            </h2>
            <button
              class="btn-tertiary"
            >
              Get Started
            </button>
          </div>
        </div>
      </div>
      <div
        class="footer"
      >
        <div
          class="footer__item-1"
        >
          <p>
             A company by Topic Talk Inglés
          </p>
          <p>
            Copyright Evaly 2021 All rights reserved
          </p>
        </div>
        <div
          class="footer__item-2"
        >
          <p
            class="footer__logo"
          >
             Evaly
          </p>
          <svg
            class="icon footer__fb-icon"
          >
            <use
              href="sprite.svg#facebook"
            />
          </svg>
          <svg
            class="icon footer__instagram-icon"
          >
            <use
              href="sprite.svg#instagram"
            />
          </svg>
        </div>
        <div
          class="footer__item-3"
        >
          <ul>
            <li>
              <a
                href="#"
              >
                Terms & Conditions
              </a>
            </li>
            <li>
              <a
                href="#"
              >
                Privacy Policy
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
